WAITLIST.s

```diff
@@ -472,7 +472,7 @@ NOQBRSM		DXCH	ARUPT
 #	LONGEXIT AND LONGEXIT+1
 #	LONGTIME AND LONGTIME+1
 #
-# *** THE FOLLOWING IS TO BE IN FIXED-FIXED AND UNSWITCHED ERASIBLE **
+# *** THE FOLLOWING IS TO BE IN FIXED-FIXED AND UNSWITCHED ERASABLE **
```

Comment:
Both are incorrect.

`ERRASIBLE` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1233.jpg).
Also missing an * in the *s after it.